lib/main.py

```diff
@@ -282,7 +283,12 @@ def init_commands(subparsers, actions_dict):
     """
     system_parser = subparsers.add_parser("system", help="Initialize a system config")
     benchpark.cmd.system.setup_parser(system_parser)
+
+    experiment_parser = subparsers.add_parser("experiment", help="Interact with experiments")
```

Comment:
Not for this PR, but we need to move the rest of the commands to `cmd/` and then turn this into a for-loop